Add docs for MyPermobil

IsakNyberg opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/95613
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4508
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

IsakNyberg committed:
add docs for MyPermobil

IsakNyberg committed:
Update mypermobil.markdown

IsakNyberg committed:
update documentation

frenck commented on source/_integrations/mypermobil.markdown:
```diff
@@ -0,0 +1,73 @@
+
+## Configuration
+
+Configure the integration by entering your email associated with your MyPermobil account, then select your region from the dropdown list. When proceeding to the next step you will receive a 6 digit code to the email you entered, this code needs to be entered in the final step. If everything works as it should, MyPermobil should be added to your list of integrations.
```
This should be provided and be clear from the config flow itself.

frenck commented on source/_integrations/mypermobil.markdown:
```diff
@@ -0,0 +1,73 @@
+
+{% include integrations/config_flow.md %}
+
+## Configuration
```
Duplicate header, this is already provided by the include above.

frenck commented on source/_integrations/mypermobil.markdown:
```diff
@@ -0,0 +1,73 @@
+
+### Battery Health
+
+The maximum amount of charge the battery can hold measured as a percentage of the battery's original capacity.
+
+### Charge Time Left
+
+A number indicating how many hours the battery needs until it is fully charged. The battery needs to be currently charging for this sensor to work.
+
+### Distance Left
+
+The distance the wheelchair can travel on its current charge. This figure is an estimate and should not be relied upon.
+
+### Indoor Drive Time
+
+The estimated number of hours of indoor drive time that is left on the current battery charge.
+
+### Battery Max Watt Hours
+
+The number of watt hours of energy stored in the battery when it is at max capacity.
+
+### Watt Hours Left
+
+The number of watt hours of energy stored in the battery on its current charge.
+
+### Full Charge Distance
+
+The distance the wheelchair can travel on a full charge. This figure is an estimate and should not be relied upon.
+
+### Distance Traveled
+
+The distance the wheelchair has traveled today.
+
+### Number of Adjustments
+
+The number of adjustments sessions today. Multiple different adjustment within a short time period are counted as a single sessions.
+
+### Record Number of Adjustments
+
+The record number of adjustments in a single day.
```
Instead of using chapters, let's just make this a list. E.g.,

```Markdown
- **Battery charge**:
  The current battery state of the wheelchair as a percentage.
- **Battery health**:
  ...
```

PS: Please note the capital letter change so it becomes sentence styled.

IsakNyberg committed:
change sensor chapters to list

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

IsakNyberg commented:
Thanks!

IsakNyberg committed:
Add notice about voice assistant linking

Users may need to activate voice assistant linking in the MyPermobil app for the config flow to work. Added a notice about this in the documentation.

IsakNyberg commented:
It was bought to my attention that for some users a linking setting needed to be changed for the integration to work, I added a note about this in the documentation.

edenhaus committed:
fix ha_release

edenhaus reviewed (dismissed):
Please fix the merge conflicts

IsakNyberg committed:
Merge branch 'next' into next

c0ffeeca7 commented on source/_integrations/mypermobil.markdown:
```diff
@@ -0,0 +1,46 @@
+ha_integration_type: hub
+---
+
+The MyPermobil integration allows you to view various sensors with information about your Permobil wheelchair, such as battery status, distance traveled, number of seating adjustments, etc. The values of the sensors may be delayed by several minutes and should not be relied on for any crucial applications. For the integration to work you must ensure that _voice assistant linking_ is activated in  ...
```
```suggestion
The **MyPermobil** integration allows you to view various sensors with information about your Permobil wheelchair. For example: battery status, distance traveled, and number of seating adjustments. The values of the sensors may be delayed by several minutes and should not be relied on for any crucial applications. For the integration to work, you must ensure that _voice assistant linking_ is activated in the MyPermobil app. To do this, open the app and navigate to **Settings** > **My Account** > **Connection Settings**.
```

c0ffeeca7 commented on source/_integrations/mypermobil.markdown:
```diff
@@ -0,0 +1,46 @@
+- **Battery charge**
+  The current battery state of the wheelchair as a percentage.
+- **Battery health**
+  The maximum amount of charge the battery can hold measured as a percentage of the battery's original capacity.
```
```suggestion
  The maximum amount of charge the battery can hold. Measured as a percentage of the battery's original capacity.
```

c0ffeeca7 commented on source/_integrations/mypermobil.markdown:
```diff
@@ -0,0 +1,46 @@
+- **Indoor drive time**
+  The estimated number of hours of indoor drive time that is left on the current battery charge.
+- **Battery max watt hours**
+  The number of watt hours of energy stored in the battery when it is at max capacity.
```
```suggestion
  The number of watt hours of energy stored in the battery when it is at maximum capacity.
```

c0ffeeca7 committed:
tiny tweaks

c0ffeeca7 commented on source/_integrations/mypermobil.markdown:
```diff
@@ -0,0 +1,46 @@
+  The distance the wheelchair has traveled today.
+- **Number of adjustments**
+  The number of adjustments sessions today. Multiple different adjustment within a short time period are counted as a single sessions.
+- **Record number of adjustments**
```
Because *Record number* is a bit unclear to me, I suggested an alternative. Please correct me if I got this wrong.

c0ffeeca7 replied:
```suggestion
- **Highest number of adjustments**
```

c0ffeeca7 commented on source/_integrations/mypermobil.markdown:
```diff
@@ -0,0 +1,46 @@
+- **Number of adjustments**
+  The number of adjustments sessions today. Multiple different adjustment within a short time period are counted as a single sessions.
+- **Record number of adjustments**
+  The record number of adjustments in a single day.
```
```suggestion
  The highest number of adjustments ever recorded in a single day.
```

IsakNyberg committed:
Update source/_integrations/mypermobil.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

IsakNyberg committed:
Update source/_integrations/mypermobil.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 reviewed (approved):
Thank you, @IsakNyberg :+1:
